manifests/init.pp

```diff
@@ -13,11 +13,15 @@
     $log_martians        = true,
     $route_filter        = true,
     $default_zone_entry  = "local firewall\n",
-    $blacklist           = ["NEW","INVALID","UNTRACKED"]
+    $blacklist           = ["NEW","INVALID","UNTRACKED"],
+    $config_test         = false,
+    $service_restart     = '/usr/sbin/shorewall check && /bin/systemctl restart shorewall',
+    $service6_restart    = '/usr/sbin/shorewall6 check && /bin/systemctl restart shorewall6',
```

Comment:
I think `config_test`, `service_restart`, and `service6_restart` ought to be parameters of the `shorewall::params` and get their default values from hiera. I can't think of any reason why these would vary other than by platform.